### ISSUE TITLE
Impossible to choose "Ctrl+Shift+|" as Global Auto type sequence

### ISSUE THREAD

**Issue by Bakuriu** [keepassxreboot/keepassxc#6223]
## Overview

I want to use Ctrl+Shift+| as Global Auto Type sequence. I'm using an Italian keyboard and this sequence  can be easily operated using only my left hand and there is 0 chances thatI mistype it or that it collides with something else.
I currently use this with KeePass and KeePassX without issues.

## Steps to Reproduce

1. Go to the settings page to choose the global auto type sequence
2. Press Ctrl, keep it pressed and press Shift than press the "\ |" key (on my keyboard it is on the left right below Esc)

## Expected Behavior

The sequece Ctrl+Shift+| appears for the Global Auto type sequence and can be saved and operated

## Actual Behavior

Instead of registering Ctrl+Shift+| as sequence it resets to empty input

Using Ctrl + Shift + !  works fine (the "1 !" key is right beside the "\ |" key on my keyboard)

## Context

KeePassXC - 2.6.4
Revision: 34a78f0
Operating System: Linux (Kubuntu)
Desktop Env: KDE Plasma
Windowing System: X11

**Comment by droidmonkey** [comment]
How are you running keepassxc? Snap? Flatpak? Native?

**Comment by droidmonkey** [comment]
Worked on Windows:
![image](https://user-images.githubusercontent.com/2809491/110064348-ecd6b000-7d3a-11eb-8ce3-b1b2b0d3b80e.png)

This is likely something we cannot overcome, we just use Qt's built-in keyboard capture widget.

**Comment by Bakuriu** [comment]
Okay, my bad I found the culprit.

I forgot that I had installed Keepass2 and I had to configure its autotype setting a global shortcut in Plasma to run `mono /usr/lib/keepass2/KeePass.exe --auto-type`, so basically when trying to input the shortcut I was inadvertently triggering this global shortcut instead.

Once I disabled the shortcut in my Settings I was able to configure KeePassXC shortcut correctly.

Sorry for wasting your time 😞
